.gitlab-ci.yml

```diff
@@ -72,22 +84,54 @@ workflow:
 .start-substrate-contracts-node-rand-extension:      &start-substrate-contracts-node-rand-extension
   - substrate-contracts-node-rand-extension --tmp --dev -lruntime::contracts=debug > /tmp/substrate-contracts-node-rand-extension.log 2>&1 &
 
+.shutdown-substrate-contracts-node:                  &shutdown-substrate-contracts-node
+  - pkill -f "substrate-contracts-node --tmp --dev"
+
+.shutdown-substrate-contracts-node-rand-extension:   &shutdown-substrate-contracts-node-rand-extension
+  - pkill -f "substrate-contracts-node-rand-extension --tmp --dev"
+
+
+# needed vars have to be "exported" on a earlier stage
+parent-vars:
+  stage:                                             declare-vars
+  <<:                                                *kubernetes-env
+  script:
+    - BRANCH="master"
+    - REPO="https://github.com/paritytech/ink.git"
+    - REPO_NAME="ink"
+    - echo ${TRGR_REF}
```

Comment:
```suggestion
    - echo ${TRIGGER_REF}
```

Reply:
`TRGR_REF` is what we receive [from the upstream](https://github.com/paritytech/ink/blob/master/.gitlab-ci.yml#L421). It can't be changed like that, sorry.